datasets/idf_utils.py

```diff
@@ -0,0 +1,56 @@
+"""
+Utilities to compute IDF scores.
+"""
+from collections import defaultdict
+
+import nltk
+nltk.download('stopwords', quiet=True)
+from nltk.corpus import stopwords
+import numpy as np
+
+
+def get_pairwise_word_to_doc_freq(sent_list_1, sent_list_2):
+    """
+    Get pairwise word to document frequency.
+    For index i, if sentence i in sent_list_1 and sentence i in sent_list_2 both
+    container word w, then w is counted only once.
+    Returns a dictionary mapping words to number of sentence pairs the word appears in.
+    """
+    word_to_doc_cnt = defaultdict(int)
+
+    sent_list_2_iter = iter(sent_list_2)
+    for s1 in sent_list_1:
```

Comment:
Why not using `for s1,s2 in zip(sent_list_1,sent_list_2): ` to loop two lists.

Reply:
Nice, forgot about it